docs/topics/auth/index.txt

```diff
@@ -60,12 +60,13 @@ startproject <startproject>`, these consist of two items listed in your
    </ref/contrib/contenttypes>`, which allows permissions to be associated with
    models you create.
 
-and two items in your :setting:`MIDDLEWARE_CLASSES` setting:
+and three items in your :setting:`MIDDLEWARE_CLASSES` setting:
 
 1. :class:`~django.contrib.sessions.middleware.SessionMiddleware` manages
    :doc:`sessions </topics/http/sessions>` across requests.
 2. :class:`~django.contrib.auth.middleware.AuthenticationMiddleware` associates
    users with requests using sessions.
+3. :class:`~django.contrib.auth.middleware.SessionAuthenticationMiddleware` logs users out of their other sessions after a password change.
```

Comment:
Please wrap lines at 79 characters as done in the above items. I've also amended the PR title to follow our commit message guidelines if you could copy it. By the way, we won't backport to 1.7 since those docs are frozen for translators (except for critical fixes). Thanks!